src/server/common/connectors/notify/notify.integration.test.js

```diff
@@ -13,7 +13,7 @@ describe('sendNotification (integration)', () => {
     const configGet = config.get.bind(config)
     const notifyConfig = {
       ...config.get('notify'),
-      timeout: 0
+      timeout: 1 // setting timeout to 1ms as 0ms means no timeout when using Wreck
```

Comment:
this looks liable to be flakey unless we introduce an explicit await in the test that definitely exceeds 1ms?

Reply:
Ah - I'm totally wrong here.  we're awaiting the eventual result of the http call